.github/workflows/check-go-version.yml

```
@@ -0,0 +1,72 @@
+on:
+  schedule:
+    # 毎週月曜日の8時15分（UTC）に実行
+    - cron: '15 8 * * 1'
+  # 手動実行も可能
+  workflow_dispatch:
+
+jobs:
+  check-latest-go:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)